config/trello.spc

```diff
@@ -0,0 +1,12 @@
+connection "trello" {
+  plugin = "trello"
+
+  # Trello API key for requests. Required.
+  # See instructions at https://developer.atlassian.com/cloud/trello/guides/rest-api/authorization/
+  # This can also be set via the TRELLO_API_KEY environment variable
+  # trello_api_key = "a25ad2e37570117c0bad72d0a711ba5af"
+
+  # Trello token for requests. Required.
+  # This can also be set via the TRELLO_TOKEN environment variable
```

Comment:
```suggestion
  # This can also be set via the `TRELLO_TOKEN` environment variable.
```

Reply:
The backticks do not appear in the `spc` files, so I don't think we usually add it here